src/test/java/seedu/planner/model/course/MajorDescriptionTest.java

```diff
@@ -73,8 +73,8 @@ public void equals() {
         // different type -> returns false
         assertFalse(cs.equals(5));
 
-        ModuleDescription cs1010_fake = new ModuleDescription("CS1010", ProgrammeRequirement.IT_PROFESSIONALISM);
-        MajorDescription csFake = new MajorDescription(Major.COMPUTER_SCIENCE, List.of(cs1010_fake));
+        ModuleDescription cs1010Fake = new ModuleDescription("CS1010", ProgrammeRequirement.IT_PROFESSIONALISM);
```

Comment:
I think can rename to `moduleDescription` or `fakeModuleDescription`

Reply:
Done.

Reply:
might want to change the `cs` and `cs1010` also